tests/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# This Dockerfile is a minimal example for a RHEL-based SSG test suite target container.
+FROM registry.access.redhat.com/rhel
```

Comment:
Why is it RHEL-based? People working on other systems don't have RHEL subscription, therefore they can't build the image using Dockerfile, therefore they can't run the test suite. Please FROM centos:7.

Reply:
I have added a CentOS Dockerfile in addition to the RHEL one..